pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -454,6 +463,7 @@ def submit_qasm(
 
         if options is not None:
             body["options"].update(options)
+        body["options"].update(self._process_circuits_options)
```

Comment:
From my understanding this allows the user to override the previously set options. Is that intended?

Reply:
Yes, but I think I actually have it the wrong way round: the options specified in the backend constructor should be overridden (potentially) by the options passed to `process_circuits()`. I will change this.

Reply:
Perfect, thank you!